src/ui/components/Timeline/Focuser.tsx

```diff
@@ -125,7 +125,17 @@ function Focuser({ editMode, setEditMode }: Props) {
     };
 
     // Stop all drag operations when the mouse leaves the window.
-    const onWindowMouseLeave = () => {
+    const onDocumentMouseLeave = (event: MouseEvent) => {
```

Comment:
Looking at the Replay, "mouseleave" was being dispatched for things other than the outer window/document. This is not how I expected it to work (or how it worked for me on MacOS + Chrome during testing) but it seems like _at least one fix_ here is to bounds check before assuming the mouse has _actually_ left the document.

Testing on Chrome, Firefox, and Safari seems good for me after this change.

@markerikson Can you verify on Windows?

Reply:
Woah, weird but good thinking!

Reply:
Seems smooth to me on both Chrome and FF!